README_JP6.md

```diff
@@ -115,8 +115,9 @@ Following steps required:
     1.	Configure Jetson AGX CSI Connector
```

Comment:
In step 4 - `4.  Copy tegra234-p3737-0000+p3701-0000-nv.dtb from host to /boot/ on Orin`, the .dtb file is stored in `/boot` folder.
But in step 6, `/boot/extlinux/extlinux.conf` - this file refers to .dtb in `/boot/dtb/` folder.

Reply:
this is generated by jetson-io script based on what it was at /boot/tegra234-p3737-0000+p3701-0000-nv.dtb